Cubical/Algebra/Module/Instances/VecFin.agda

```diff
@@ -0,0 +1,46 @@
+{-# OPTIONS --safe #-}
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.SIP
+
+open import Cubical.Data.Nat using (ℕ)
+open import Cubical.Data.FinData
+
+open import Cubical.Reflection.RecordEquiv
+
+open import Cubical.Algebra.Monoid
+open import Cubical.Algebra.Ring
+open import Cubical.Algebra.Group
+open import Cubical.Algebra.Semigroup
+open import Cubical.Algebra.AbGroup
+open import Cubical.Algebra.Module
+
+module Cubical.Algebra.Module.Instances.VecFin {ℓ} (R : Ring ℓ) {n : ℕ} where
```

Comment:
Please make ```module Cubical.Algebra.Module.Instances.VecFin where``` the second line of the file and use an anonymous module here, to introduce the parameters, i.e. ```module _ {ℓ} (R : Ring ℓ) {n : ℕ} where``` (and intend what follows).

Reply:
I have already fixed that.